### ISSUE TITLE
Issues with some pref/sref tags

### ISSUE THREAD

**Issue by LukeButtersFunnelback** [w3c/aria#580]
Hi
When reading https://github.com/w3c/aria/blob/master/aria/aria.html i noticed that:
* https://github.com/w3c/aria/blob/master/aria/aria.html#L1157 is wrong as `aria-expanded` is a state and so should be in a `<sref>` tag.
* https://github.com/w3c/aria/blob/master/aria/aria.html#L1158 is wrong as `aria-pressed` is a state and so should be in a `<sref>` tag
* https://github.com/w3c/aria/blob/master/aria/aria.html#L3150 is wrong as `aria-expanded` is a state and so should be in a `<sref>` tag.
* https://github.com/w3c/aria/blob/master/aria/aria.html#L4558 is wrong as `aria-checked` is a state and so should be in a `<sref>` tag.

hmm maybe I shouldn't be trying to parse that file?

**Comment by Joanmarie** [comment]
Should be fixed. If I missed any, please reopen. Thanks!

**Comment by LukeButtersFunnelback** [comment]
great thank you
